keadm/cmd/keadm/app/cmd/cloud/gettoken.go

```diff
@@ -74,7 +75,8 @@ func queryToken(namespace string, name string, kubeConfigPath string) ([]byte, e
 
 // showToken prints the token
 func showToken(data []byte, out io.Writer) error {
-	_, err := out.Write(data)
+	token := append(data, byte('\n'))
+	_, err := out.Write(token)
```

Comment:
thanks for the fixing! @ESWZY 
how about `_, err := fmt.Fprintln(out, data)`?

Reply:
Sounds great!